photon-core/src/main/java/org/photonvision/vision/camera/USBCameras/USBCameraSource.java

```diff
@@ -113,7 +112,7 @@ protected GenericUSBCameraSettables createSettables(
         GenericUSBCameraSettables settables;
 
         if (quirks.hasQuirk(CameraQuirk.LifeCamControls)) {
-            if (RuntimeDetector.isWindows()) {
+            if (System.getProperty("os.name").startsWith("Windows")) {
```

Comment:
This feels hacky. Let's keep some form of RuntimeDetector around.

Reply:
This is what RuntimeDetector did internally. We could add this functionality to platform?

Reply:
This should be something like Platform getcurrentplatform ostype = windows right?